src/sidebar/util/test/build-thread-test.js

```diff
@@ -475,16 +481,14 @@ describe('sidebar/util/build-thread', function () {
         },
       ];
       const thread = createThread(fixture, {
-        sortCompareFn: function (a, b) {
-          return a.id < b.id;
-        },
+        sortCompareFn: (a, b) => b.annotation.id - a.annotation.id,
```

Comment:
This code works because of JavaScript's handling of string subtraction, but this wouldn't work if the data were more realistic. `a.id.localeCompare(b.id)` would work however.

Reply:
Updated.